main.py

```diff
@@ -8,5 +9,13 @@ def run_example():
     second = input("b: ")
     print(example(first, second))
 
+
+def run_new_avg():
+    """Run new_avg function."""
+    arr = input("Array: ")
+    navg = int(input("Expected average: "))
```

Comment:
here you should add try/except block for input

Reply:
Done.